sssom/constants.py

```diff
@@ -177,3 +166,59 @@ class SchemaValidationType(str, Enum):
     SchemaValidationType.JsonSchema,
     SchemaValidationType.PrefixMapCompleteness,
 ]
+
+
+class SSSOMSchemaView(object):
+    """
+    SchemaView class from linkml which is instantiated when necessary.
+
+    Reason for this: https://github.com/mapping-commons/sssom-py/issues/322
+    Implemented via PR: https://github.com/mapping-commons/sssom-py/pull/323
+    """
+
+    _view = None
+    _dict = None
+
+    def __new__(cls):
+        """Create a instance of the SSSOM schema view if non-existent."""
+        if not hasattr(cls, "instance"):
+            cls.instance = super(SSSOMSchemaView, cls).__new__(cls)
+            return cls.instance
+
+    @property
+    def view(self) -> SchemaView:
+        """Return SchemaView object."""
+        if self._view is None:
+            self._view = SchemaView(SCHEMA_YAML)
+        return self._view
+
+    @property
+    def dict(self) -> dict:
+        """Return SchemaView as a dictionary."""
+        if self._dict is None:
+            self._dict = schema_as_dict(self.view.schema)
+        return self._dict
+
+    @property
+    def mapping_slots(self) -> List[str]:
+        """Return list of mapping slots."""
+        return self.dict["classes"]["mapping"]["slots"]
```

Comment:
just making a non-blocking note here. We will later refactor this, there is zero need for a dict or for doing dict lookups. the entire point of SV is to abstract away from this low level representation!

however, we now have a buffering layer with SSSOMSchemaView, so this is something we can do transparently later on!